### ISSUE TITLE
Suggestion - Add log trimming to docker-compose

### ISSUE THREAD

**Issue by alecisec** [PromoFaux/MattermostRSS#12]
I've modified my docker-compose.yml to add the following:

    logging:
      driver: "json-file"
      options:
        max-size: "200k"
        max-file: "10"

some variation on this might be appropriate for the example yml to prevent logs becoming unmanageable (STDERR output isn't capped and bloats the container after long running periods)

**Comment by PromoFaux** [comment]
Thanks, I'll look into this. The `docker-compose.yml` that is part of the project now is just what is generated by visual studio when you choose to add docker support to the project. Personally, I only use it when building and debugging.

Are you using the docker compose to build and deploy the image locally?

**Comment by alecisec** [comment]
Yup, I'm using the compose script to build it up each time the codebase gets updated, just copying the secrets.json across to the new container, so the 'last post' timestamps and increased logging verbosity are god-sends thanks.
